src/main/components/Report.js

```diff
@@ -1,64 +1,75 @@
-import React, { Component } from 'react';
-import { Row, Col } from 'react-flexbox-grid';
+import React, {Component} from 'react';
+import {Row, Col} from 'react-flexbox-grid';
 import TalkCard from './TalkCard';
+import SpeakerCard from './SpeakerCard';
 import ReportStats from './ReportStats';
 
-
 import testImage from '../../test/snapshot/images/test-image.jpeg';
 
 const talkDetail = {
-  title: 'Welcome to Devoxx 2017',
-  description: 'Join the organisers of Devoxx UK and great keynote ' +
-  'speakers for inspring stories in 20 minute segments.',
-  rating: 4,
-  topTracks: [
-    'Java',
-    'Devoxx',
-    'Spring',
-  ],
-  notes: 'Lorem ipsum dolor sit amet, everti quaestio mel ea. Ex eos ' +
-  'volutpat qualisque. Sale tantas cotidieque quo ut, ad nostro consectetuer' +
-  ' nec. Feugiat qualisque quo an. Labores officiis te nam.',
-  review: {
-    name: 'Test User',
-    comment: 'Great session, thanks for organising. Looking forward to the next one!',
-    image: testImage
-  }
+    title: 'Welcome to Devoxx 2017',
+    description: 'Join the organisers of Devoxx UK and great keynote ' +
+    'speakers for inspring stories in 20 minute segments.',
+    rating: 4,
+    topTracks: [
+        'Java',
+        'Devoxx',
+        'Spring',
+    ],
+    notes: 'Lorem ipsum dolor sit amet, everti quaestio mel ea. Ex eos ' +
+    'volutpat qualisque. Sale tantas cotidieque quo ut, ad nostro consectetuer' +
+    ' nec. Feugiat qualisque quo an. Labores officiis te nam.',
+    review: {
+        name: 'Test User',
+        comment: 'Great session, thanks for organising. Looking forward to the next one!',
+        image: testImage
+    }
 };
 
+const speakerDetail = {
```

Comment:
Shouldn't be here?! @sam-eade - all data should be at 'App.js' level and passed down via props